src/index.scss

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:thin,extra-light,light,100,200,300,400,500,600,700,800');
+
+// Fonts
+$montserrat: 'Montserrat', sans-serif;
+$roboto: 'Roboto', sans-serif;
+
+// Colors
+$azure: #0290ff;
+$coolBlue: #4386bf;
+$greyBorder: #e8e8e8;
+$pinkishGrey: #c4c4c4;
+$warmGrey: #888;
+$paleGrey: #fafafa;
+
+// General padding
+$generalPadding: 0 4rem;
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: $paleGrey;
+  margin: 0;
+  font-family: $montserrat;
+}
+
+.App {
+  text-align: center;
+}
+
+button {
+  font-family: $roboto;
+}
+
+//NAVBAR
+header {
+  background-color: #fff;
+}
+
+.navbar {
+  height: 5.938rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  margin-bottom: 2.313rem;
+}
+
+.nav-title {
+  color: $azure;
+  font-size: 1.875rem;
+}
+
+.books,
+.categories {
+  font-size: 0.813rem;
+  text-decoration: none;
+}
+
+.books {
+  margin-right: 2.563rem;
+  color: #121212;
+}
+
+.categories {
+  color: $warmGrey;
+}
+
+.oval {
+  width: 2.813rem;
+  height: 2.813rem;
+  padding: 0.875rem 0.9rem 0.875rem 0.875rem;
+  border: 1px solid $greyBorder;
+  border-radius: 2rem;
+  margin-left: 20.5rem;
+}
+
+.mask {
+  text-align: center;
+  width: 1.363rem;
+  height: 1.363rem;
+  color: $azure;
+  margin-right: 0.9rem;
+}
+
+//BOOKITEM
+.bookItem {
+  display: flex;
+  background-color: #fff;
+  height: 10.625rem;
+  border: 1px solid $greyBorder;
+  margin: 0.913rem 6.25rem;
+  border-radius: 4px;
+  justify-content: space-between;
+  padding: 1rem 6.188rem 1.625rem 1rem;
+  align-items: center;
+}
+
+.book {
+  float: inline-start;
+  text-align: left;
+}
+
+.category {
+  color: $warmGrey;
+}
+
+.bookTitle {
+  color: #121212;
+  font-family: $roboto;
+  font-weight: bold;
+  font-size: 1.575rem;
+}
+
+.bookAuthor,
+.onBook,
+.remove {
+  color: $coolBlue;
+  font-weight: 300;
+  font-size: 0.875rem;
+  font-family: $roboto;
+}
+
+.onBook,
+.remove {
+  background-color: #fff;
+  border: none;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+}
+
+.remove {
+  border-left: 1.5px solid $greyBorder;
+  border-right: 1px solid $greyBorder;
+  padding: 0 0.9rem;
+}
+
+//PROGRESS
+
+.chapterHeading {
+  color: $warmGrey;
+  font-family: $roboto;
+  font-size: 0.913rem;
+  font-weight: 300;
+  opacity: 0.9;
+  margin-left: -2.5rem;
+}
+
+.chapter {
+  color: #121212;
+  font-weight: 300;
+  font-family: $roboto;
+  font-size: 1rem;
+  margin-left: -6.5rem;
+}
+
+.progress {
+  background-color: $azure;
+  color: #fff;
+  font-weight: 300;
+  font-size: 0.913rem;
+  border: none;
+  border-radius: 3px;
+  text-align: center;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+}
+
+.comment {
+  margin-left: -0.45rem;
+}
+
+//ADD BOOK
+
+div .line {
+  width: 70rem;
+  height: 0.125rem;
+  border: 1px solid $greyBorder;
+  margin: 2.5rem 4rem 1.813rem 5.9rem;
+}
+
+.adding {
+  display: flex;
```

Comment:
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [ ] This is just an example if you use duplicate styles please combine them.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)